session/session.go

```diff
@@ -825,12 +825,92 @@ func (s *session) SetGlobalSysVar(name, value string) error {
 	if err != nil {
 		return errors.Trace(err)
 	}
+
+	switch name {
+	case variable.TiDBMaxChunkSize:
+		return s.persistVarInConstraint(name, sVal, variable.TiDBInitChunkSize, func(s *session, val string) (string, error) {
+			newMaxChunkSize, err := strconv.Atoi(sVal)
+			if err != nil {
+				return "", errors.Trace(err)
+			}
+			initChunkSize, err := strconv.Atoi(val)
+			if err != nil {
+				return "", errors.Trace(err)
+			}
+			if initChunkSize > newMaxChunkSize {
+				return "", errors.Errorf("Try to set TiDB MaxChunkSize to %d small than InitChunkSize %d, please change InitChunkSize ahead",
```

Comment:
how about "tidb-max-chunk-size(%d) cannot be smaller than tidb-init-chunk-size(%d), please decrease tidb-init-chunk-size first"?

Reply:
Advice from @CaitinChen 
"tidb_max_chunk_size" should be no less than "tidb_init_chunk_size". But %d is less than %d. DefaultChunkSize %d will be used forcely.